src/main/java/duke/common/Messages.java

```diff
@@ -0,0 +1,12 @@
+package duke.common;
+
+public class Messages {
+    public static final String MESSAGE_GREETING = "Hello! I'm Duke";
+    public static final String MESSAGE_INQUIRY = "What can I do for you?";
+    public static final String MESSAGE_TASK_LISTING_HEADER = "Here are the tasks in your list:";
+    public static final String MESSAGE_ADD_TASK_ACKNOWLEDGEMENT = "Got it. I've added this task: ";
+    public static final String MESSAGE_DELETE_TASK_ACKNOWLEDGEMENT = "Noted. I've removed this task:";
+    public static final String MESSAGE_MARK_TASK_ACKNOWLEDGEMENT = "Nice! I've marked this task as done:";
+    public static final String MESSAGE_UNMARK_TASK_ACKNOWLEDGEMENT = "OK, I've marked this task as not done yet:";
+    public static final String MESSAGE_FAREWELL = "Bye. Hope to see you again soon!";
+}
```

Comment:
Might want to make this an `enum` like `ErrorMessages`.

Reply:
Suggestion adopted, thanks!